ERROR - 2023-04-12 07:19:38 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->store()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-12 07:19:38 --> Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (:nama:, :username:, :password:, :level:, :kode_pendaftar:, :alamat:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DatauserController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->store()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 07:19:42 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->store()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-12 07:19:42 --> Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (:nama:, :username:, :password:, :level:, :kode_pendaftar:, :alamat:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DatauserController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->store()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 07:19:59 --> mysqli_sql_exception: Duplicate entry '2' for key 'PRIMARY' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->store()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-12 07:19:59 --> Duplicate entry '2' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'raka\', \'rakasahalgay\', \'rakagay\', \'pengguna\', \'2\', \'Kota Pontianak\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (:nama:, :username:, :password:, :level:, :kode_pendaftar:, :alamat:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DatauserController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->store()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 07:23:05 --> mysqli_sql_exception: Duplicate entry '1' for key 'PRIMARY' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->store()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-12 07:23:05 --> Duplicate entry '1' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'P\', \'nikolas\', \'bona12345\', \'bengkel\', \'1\', \'Jl.Ampera\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'P\', \'nikolas\', \'bona12345\', \'bengkel\', \'1\', \'Jl.Ampera\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'P\', \'nikolas\', \'bona12345\', \'bengkel\', \'1\', \'Jl.Ampera\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (:nama:, :username:, :password:, :level:, :kode_pendaftar:, :alamat:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DatauserController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->store()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 07:33:21 --> mysqli_sql_exception: Unknown column 'id' in 'field list' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `id`, `n...', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `n...')
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `n...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(189): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(82): Agoenxz21\Datatables\Datatable->getResult()
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(21): Agoenxz21\Datatables\Datatable->draw()
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->all()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-12 07:33:21 --> Unknown column 'id' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('SELECT `id`, `nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`
FROM `data_user`
 LIMIT 10', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `id`, `nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`
FROM `data_user`
 LIMIT 10')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `id`, `nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`
FROM `data_user`
 LIMIT 10')
 4 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`
FROM `data_user`
 LIMIT 10', [], false)
 5 SYSTEMPATH\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
 6 VENDORPATH\agoenxz21\datatables\src\Datatable.php(189): CodeIgniter\Model->__call('get', [])
 7 VENDORPATH\agoenxz21\datatables\src\Datatable.php(82): Agoenxz21\Datatables\Datatable->getResult()
 8 APPPATH\Controllers\DatauserController.php(21): Agoenxz21\Datatables\Datatable->draw()
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->all()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 07:34:05 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bengkelakiong' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bengkelakiong', 3306, '', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(189): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(82): Agoenxz21\Datatables\Datatable->getResult()
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(21): Agoenxz21\Datatables\Datatable->draw()
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->all()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bengkelakiong' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(189): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(82): Agoenxz21\Datatables\Datatable->getResult()
#6 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(21): Agoenxz21\Datatables\Datatable->draw()
#7 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->all()
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-04-12 07:34:05 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bengkelakiong'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`
FROM `data_user`
 LIMIT 10', [], false)
 3 SYSTEMPATH\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
 4 VENDORPATH\agoenxz21\datatables\src\Datatable.php(189): CodeIgniter\Model->__call('get', [])
 5 VENDORPATH\agoenxz21\datatables\src\Datatable.php(82): Agoenxz21\Datatables\Datatable->getResult()
 6 APPPATH\Controllers\DatauserController.php(21): Agoenxz21\Datatables\Datatable->draw()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->all()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 07:47:36 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bengkelakiong' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bengkelakiong', 3306, '', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(189): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(82): Agoenxz21\Datatables\Datatable->getResult()
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(21): Agoenxz21\Datatables\Datatable->draw()
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->all()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bengkelakiong' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `n...', Array, false)
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(189): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\projecttesting\vendor\agoenxz21\datatables\src\Datatable.php(82): Agoenxz21\Datatables\Datatable->getResult()
#6 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(21): Agoenxz21\Datatables\Datatable->draw()
#7 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->all()
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-04-12 07:47:36 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'bengkelakiong'
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT `id`, `nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`
FROM `data_user`
 LIMIT 10', [], false)
 3 SYSTEMPATH\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
 4 VENDORPATH\agoenxz21\datatables\src\Datatable.php(189): CodeIgniter\Model->__call('get', [])
 5 VENDORPATH\agoenxz21\datatables\src\Datatable.php(82): Agoenxz21\Datatables\Datatable->getResult()
 6 APPPATH\Controllers\DatauserController.php(21): Agoenxz21\Datatables\Datatable->draw()
 7 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->all()
 8 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
 9 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 12:23:59 --> mysqli_sql_exception: Duplicate entry '123' for key 'PRIMARY' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->store()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-12 12:23:59 --> Duplicate entry '123' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'ppp\', \'pengguna\', \'123\', \'Pontianak\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'ppp\', \'pengguna\', \'123\', \'Pontianak\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'ppp\', \'pengguna\', \'123\', \'Pontianak\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (:nama:, :username:, :password:, :level:, :kode_pendaftar:, :alamat:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DatauserController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->store()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
CRITICAL - 2023-04-12 12:24:39 --> Deletes are not allowed unless they contain a "where" or "like" clause.
in SYSTEMPATH\Database\BaseBuilder.php on line 2767.
 1 SYSTEMPATH\Model.php(456): CodeIgniter\Database\BaseBuilder->delete()
 2 SYSTEMPATH\BaseModel.php(1065): CodeIgniter\Model->doDelete('0', false)
 3 APPPATH\Controllers\DatauserController.php(63): CodeIgniter\BaseModel->delete('0')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->delete()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
CRITICAL - 2023-04-12 12:24:41 --> Deletes are not allowed unless they contain a "where" or "like" clause.
in SYSTEMPATH\Database\BaseBuilder.php on line 2767.
 1 SYSTEMPATH\Model.php(456): CodeIgniter\Database\BaseBuilder->delete()
 2 SYSTEMPATH\BaseModel.php(1065): CodeIgniter\Model->doDelete('0', false)
 3 APPPATH\Controllers\DatauserController.php(63): CodeIgniter\BaseModel->delete('0')
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->delete()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 12:25:36 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->store()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-12 12:25:36 --> Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'p\', \'pengguna\', \'p\', \'p\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'p\', \'pengguna\', \'p\', \'p\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'p\', \'pengguna\', \'p\', \'p\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (:nama:, :username:, :password:, :level:, :kode_pendaftar:, :alamat:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DatauserController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->store()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
ERROR - 2023-04-12 12:28:34 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'bengkelakiong' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'root', '', 'bengkelakiong', 3306, '', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\projecttesting\spark(97): CodeIgniter\CLI\Console->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'bengkelakiong' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1408): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(1427): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(768): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MigrationRunner.php(168): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CLI\Commands.php(65): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CLI\Console.php(37): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\projecttesting\spark(97): CodeIgniter\CLI\Console->run()
#10 {main}
ERROR - 2023-04-12 12:32:44 --> mysqli_sql_exception: Duplicate entry 'p' for key 'PRIMARY' in C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `da...', 0)
#1 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `da...')
#2 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `da...')
#3 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `da...', Array, false)
#4 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\projecttesting\app\Controllers\DatauserController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\DatauserController->store()
#9 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
#10 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\projecttesting\public\index.php(67): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\projecttesting\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2023-04-12 12:32:44 --> Duplicate entry 'p' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'p\', \'pengguna\', \'p\', \'p\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'p\', \'pengguna\', \'p\', \'p\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (\'p\', \'p\', \'p\', \'pengguna\', \'p\', \'p\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `data_user` (`nama`, `username`, `password`, `level`, `kode_pendaftar`, `alamat`) VALUES (:nama:, :username:, :password:, :level:, :kode_pendaftar:, :alamat:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\DatauserController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\DatauserController->store()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\DatauserController))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\projecttesting\\public\\index.php')
